dusk-abi/src/lib.rs

```diff
@@ -238,6 +248,31 @@ pub fn poseidon_hash(messages: Vec<BlsScalar>) -> BlsScalar {
     BlsScalar::from_bytes(&result).expect("A proper BlsScalar")
 }
 
+/// Verify a PLONK proof given the Proof, VerifierKey and PublicInputs
+pub fn verify_proof(
+    proof: Vec<u8>,
+    vk: Vec<u8>,
+    label: Vec<u8>,
```

Comment:
I believe we also need the crate name here, yes?

Reply:
The label includes everything. Makes it much easier TBH..

Reply:
I think we agreed *temporary* to just use the label without the crate inside. Since currently we don't have label duplicated across crates, so they're already unique.

We'll have to remove that approach anyway so it shouldn't a big deal currently for just the transfer and bid contract.